Sources/PointFree/Database.swift

```diff
@@ -414,6 +422,35 @@ private func fetchUsersSubscribed(to newsletter: Database.EmailSetting.Newslette
   )
 }
 
+private func fetchUsersToWelcome(fromWeeksAgo weeksAgo: Int) -> EitherIO<Error, [Database.User]> {
+  let daysAgo = weeksAgo * 7
+  return rows(
+    """
+    SELECT
+        "users"."email",
+        "users"."episode_credit_count",
+        "users"."github_user_id",
+        "users"."github_access_token",
+        "users"."id",
+        "users"."is_admin",
+        "users"."name",
+        "users"."subscription_id"
+    FROM
+        "email_settings"
+        LEFT JOIN "users" ON "email_settings"."user_id" = "users"."id"
+        LEFT JOIN "subscriptions" on "users"."id" = "subscriptions"."user_id"
+    WHERE
+        "email_settings"."newsletter" = $1
+        AND "users"."created_at" BETWEEN CURRENT_DATE - INTERVAL '\(daysAgo) DAY'
+        AND CURRENT_DATE - INTERVAL '\(daysAgo - 1) DAY'
+        AND "users"."subscription_id" IS NULL
+        AND "subscriptions"."user_id" IS NULL;
+    """,
+    // TODO: protect against users that own a sub but are not using a sub
```

Comment:
looks like maybe this TODO is done? to filter `"subscriptions"."user_id" IS NULL` means that subscription owners aren't included?

can you confirm?